frontend/.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
All our out .eslintrc are in yaml format. Also, we already have one in browser/ and one in public/javascript/ as a starting point.

Reply:
Ok, will aggregate.